src/coreclr/vm/ceeload.h

```diff
@@ -87,6 +87,8 @@ class JITInlineTrackingMap;
 #define NATIVE_SYMBOL_READER_DLL W("Microsoft.DiaSymReader.Native.arm.dll")
 #elif defined(HOST_ARM64)
 #define NATIVE_SYMBOL_READER_DLL W("Microsoft.DiaSymReader.Native.arm64.dll")
+#elif defined(HOST_LOONGARCH64)
+#define NATIVE_SYMBOL_READER_DLL W("Microsoft.DiaSymReader.Native.loongarch64.dll")
```

Comment:
Wow. Does this part of standard Nuget package, or this is something custom?

Reply:
And frankly, here I just add LoongArch64's supporting while don't understand it's usage.

Reply:
This DLL is Windows only as far as I aware. Not sure why this change is even needed. Better wait if this PR can be make green based on other comments in sibling PRs

Reply:
> This DLL is Windows only as far as I aware. Not sure why this change is even needed. Better wait if this PR can be make green based on other comments in sibling PRs

I will revert on this PR.
Thanks.